hack/vendor.sh

```diff
@@ -13,6 +13,7 @@ clone git github.com/gorilla/context 14f550f51a
 clone git github.com/gorilla/mux e444e69cbd
 clone git github.com/kr/pty 5cf931ef8f
 clone git github.com/microsoft/hcsshim f674a70f1306dbe20b3a516bedd3285d85db60d9
+clone git github.com/mattn/go-sqlite3 b4142c444a8941d0d92b0b7103a24df9cd815e42
```

Comment:
This list is supposed to be sorted, so I think this one actually should be up above `microsoft/hcsshim`.